workflow/metrics/build_info.go

```diff
@@ -0,0 +1,33 @@
+package metrics
+
+import (
+	"context"
+
+	"github.com/argoproj/argo-workflows/v3"
+)
+
+func addBuildInfo(ctx context.Context, m *Metrics) error {
+	const nameBuildInfo = `controller_build_info`
+	err := m.createInstrument(int64Counter,
+		nameBuildInfo,
+		"Build Information for the Argo Workflows Controller",
+		"{unused}",
+		withAsBuiltIn(),
+	)
+	if err != nil {
+		return err
+	}
+
+	version := argo.GetVersion()
+	m.addInt(ctx, nameBuildInfo, 1, instAttribs{
+		{name: labelBuildVersion, value: version.Version},
```

Comment:
is there any way to add descriptions to attributes in OTel?

Reply:
No, that isn't possible.